fdbserver/GlobalTagThrottler.actor.cpp

```diff
@@ -325,14 +332,34 @@ class GlobalTagThrottlerImpl {
 		return limitingCostForTag / averageTransactionCost.get();
 	}
 
-	// Return the limiting transaction rate, aggregated across all storage servers
+	// Return the limiting transaction rate, aggregated across all storage servers.
+	// The limits from the worst maxFallingBehind zones are
+	// ignored
 	Optional<double> getLimitingTps(TransactionTag tag) const {
-		Optional<double> result;
-		for (const auto& [id, _] : throttlingRatios) {
-			auto const targetTpsForSS = getLimitingTps(id, tag);
-			result = getMin(result, targetTpsForSS);
+		// TODO: The algorithm for ignoring the worst zones can be made more efficient
+		std::map<Optional<Standalone<StringRef>>, double> zoneIdToLimitingTps;
```

Comment:
Is there a reason this needs to be `std::map` instead of `std::unordered_map`? I don't think we're using ordering by `zoneId` anywhere? The latter would make it more efficient.